python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -624,6 +646,13 @@ def terminate_scheduled_nodes(self):
         for node in self.nodes_to_terminate:
             self.node_tracker.untrack(node)
             self.prom_metrics.stopped_nodes.inc()
+            # Clean up replicas to delete
+            tags = self.provider.node_tags(node)
+            if TAG_RAY_REPLICA_INDEX in tags:
+                replica_id = tags[TAG_RAY_REPLICA_INDEX]
+                self.replicas_to_nodes[replica_id].remove(node)
```

Comment:
`self` seems to refer to `StandardAutoscaler` here. Does it have `replicas_to_node`?

Reply:
I added in an `assert` statement, but `replicas_to_node` is initialized in the constructor so it should have it.

Reply:
Done in 627fcb2.